umap/static/umap/base.css

```diff
@@ -500,6 +500,17 @@ i.info {
 .umap-edit-actions li:hover {
     background-color: #353c3e;
 }
+.permissions-panel,
```

Comment:
:sweat_smile: Sorry i do not understand your question.

In fact, i changed the global popup padding from 20px to 10px.
To keep the total padding to 20px, i was obliged to add it in the différents div included inside the popup: permission-panel and so on.

And sadly for the description linked to the marker, there is not a single div, but 4 differents div (red, green, blue and orange borders below). And I want to apply the 10px to the orange div, because this is the efffective div that contains the content and containing the scroll bar.
![image](https://user-images.githubusercontent.com/6597721/78872856-2da24800-7a4a-11ea-9e14-63cd4cd48c8e.png)

Here is what happen with my code and a scroll bar:
![image](https://user-images.githubusercontent.com/6597721/78872977-60e4d700-7a4a-11ea-9181-a0a8606342b0.png)

Without my code:
![image](https://user-images.githubusercontent.com/6597721/78873173-b4efbb80-7a4a-11ea-8815-b6f54067f3c3.png)

This is why i moved the 10px padding closer to the content :wink: